x-pack/plugins/uptime/public/lib/alert_types/monitor_status_title.tsx

```diff
@@ -4,14 +4,23 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import React from 'react';
-import { useSelector } from 'react-redux';
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
 import { FormattedMessage } from '@kbn/i18n/react';
 import { EuiFlexGroup, EuiFlexItem, EuiLoadingSpinner, EuiText } from '@elastic/eui';
-import { snapshotDataSelector } from '../../state/selectors';
+import { snapshotDataSelector, esKuerySelector } from '../../state/selectors';
+import { getSnapshotCountAction } from '../../state/actions';
 
 export const MonitorStatusTitle = () => {
   const { count, loading } = useSelector(snapshotDataSelector);
+  const esKuery = useSelector(esKuerySelector);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(
+      getSnapshotCountAction({ dateRangeStart: 'now-15m', dateRangeEnd: 'now', filters: esKuery })
+    );
+  }, [dispatch, esKuery]);
```

Comment:
there is some room for improvement here once it's opened in alerting app, filter doens't get's updated when user change filters in the alert dialogue.

Reply:
actually it's not updating the monitor count even inside uptime app.